server/src/controllers/christmas-tree.es6

```diff
@@ -441,4 +445,56 @@ christmasTree.cancelOne = (req, res) => {
       res.status(404).send();
     });
 };
+
+christmasTree.getPermits = (req, res) => {
```

Comment:
are all the xmas tree permit endpoints reflected in Swagger?

Reply:
added a task on 285